test/e2e/drc/drc_test.go

```diff
@@ -1,81 +0,0 @@
-package drc_test
-
-import (
-	"time"
-
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
-
-	"github.com/ucloud/redis-cluster-operator/test/e2e"
-)
-
-var (
-	goredis *e2e.GoRedis
-	dbsize  int64
-	err     error
-)
-
-var _ = Describe("DistributedRedisCluster CRUD", func() {
```

Comment:
is this not the main e2e tests file? why do we have it deleted? It covers a lot of use-cases